ONSdigital/florence
# Move prerelease, remove version reloading, fix URI check

### What

- Move the prerelease button to the bottom of the editor
- Remove the version reloading functionality
- Fix the URI check when editing content

### How to review

- Check that the prerelease button has moved
- Check that version reloading doesn't happen
- Check that the URI test works

### Who can review

Anyone except @ian-kent
